lib/beaker/dsl/install_utils/aio_defaults.rb

```diff
@@ -17,13 +17,13 @@ module AIODefaults
           },
           'windows' => { #windows
             'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin',
-            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
+            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/puppet/bin',
```

Comment:
One thing that you could do is append the path, but I'll have to think a bit about whether or not that's a bad idea.

My preference would be to not have to make any Beaker adjustments, and to keep everything consistent with what we were shipping.  There might be tests scattered about (in other projects) that don't use `privatebindir`, and instead hardcode these paths.  That would then require updating the existing tests in addition to Beaker.